ConsoleApp2/Program.cs

```diff
@@ -54,6 +54,12 @@ static void Main(string[] args)
                         Console.WriteLine($"The result is: {r}");
                         break;
                     case "5":
+                        Console.WriteLine("Square root");
+                        Console.WriteLine("Enter the number you want to get the square root of");
+                        double sqrt = Convert.ToInt32(Math.Sqrt(Convert.ToDouble(Console.ReadLine())));
```

Comment:
why did u use Convert. twise?

Reply:
![image](https://user-images.githubusercontent.com/70525597/91960143-330e7080-ed12-11ea-935c-7c84c50ebd17.png)

Reply:
@SMetelitsa  Math.Sqrt method takes an argument with double type so I had to convert to double type to pass it

Reply:
@SMetelitsa also I noticed that will be always as an integer value so it needs to be fixed as well

Reply:
u r right